qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/presenters/ListAffiliationsPresenter.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.portal.qoffer2.web.presenters
+
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.portal.portlet.customers.affiliation.list.ListAffiliationsOutput
+import life.qbic.portal.qoffer2.web.viewmodel.ViewModel
+
+/**
+ * <short description>
+ *
+ * <detailed description>
+ *
+ * @since: <versiontag>
```

Comment:
```suggestion
 * This class acts as a presenter for the ListAffiliation use case. It takes care of reflecting the updated information in the view model
 *
 * @since: 1.0.0
```